Classification/Classification.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+export default function UserName({ setUserInfo }) {
```

Comment:
Classification has UserName? The filename should match the Component its exporting